pkg/controllers/dynakube/oneagent/oneagent_reconciler.go

```diff
@@ -12,7 +12,10 @@ import (
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/connectioninfo"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/deploymentmetadata"
 	"github.com/Dynatrace/dynatrace-operator/pkg/controllers/dynakube/oneagent/daemonset"
-	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/hasher"
+	"github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/configmap"
+	daemonset2 "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/daemonset"
```

Comment:
```suggestion
	kodaemonset "github.com/Dynatrace/dynatrace-operator/pkg/util/kubeobjects/daemonset"
```
or any other name, 2 is a bit confusing.

Reply:
this seems to have been done automatically by goimports, of course i would not name it like that :sweat_smile: Thanks for pointing out! I renamed this as well as all other occurrences. done in https://github.com/Dynatrace/dynatrace-operator/pull/2336/commits/b58b302251d3905a66492b1b0993d5ee3d7f2669